### ISSUE TITLE
Generate SV predictions in VCF format

### ISSUE THREAD

**Issue by lsantuari** [GooglingTheCancerGenome/CNN#64]
Convert the SV predictions from BEDPE to VCF format (version 4.3).
_sv-callers_ VCF output (version 4.3).

- [x] DEL
- [x] INS
- [x] DUP
- [x] INV
- [x] CTX

**Comment by arnikz** [comment]
@lsantuari: based on commit 2c0226066edf1d9d5494f1378dce53195737eb2d

I can't reproduce your successful run on the HPC cluster (Spider). Same jobs failed as indicated by Travis CI.

### local

https://travis-ci.org/github/GooglingTheCancerGenome/sv-channels/jobs/742890329#L5090

### GridEngine

_jobid_ - _jobname_
31 - bedpe_to_vcf-split_reads-kfold
32 - bedpe_to_vcf-split_reads-chrom
33 - bedpe_to_vcf-gridss-kfold
34 - bedpe_to_vcf-gridss-chrom
https://travis-ci.org/github/GooglingTheCancerGenome/sv-channels/jobs/742890330#L19660-L19760

### Slurm

_jobid_ - _jobname_
22 - add_win_channels-split_reads-DEL
23 - add_win_channels-gridss-DEL
24 - train-DEL-split_reads
25 - train-DEL-split_reads
26 - train-DEL-gridss
27 - train-DEL-gridss
32 - bedpe_to_vcf-split_reads-kfold
33 - bedpe_to_vcf-split_reads-chrom
34 - bedpe_to_vcf-gridss-kfold
35 - bedpe_to_vcf-gridss-chrom
https://travis-ci.org/github/GooglingTheCancerGenome/sv-channels/jobs/742890331

**Spider**

```
$ cd $MYHOME/sv-channels/
$ cat scripts/utils/jobs.json |jq -r '.statuses[]|select(.state=="FAILED")|[.jobIdentifier,.name]|@csv'
"196903","add_win_channels-split_reads-DEL"
"196904","add_win_channels-gridss-DEL"
"196905","train-DEL-split_reads"
"196906","train-DEL-split_reads"
"196907","train-DEL-gridss"
"196908","train-DEL-gridss"
"196913","bedpe_to_vcf-split_reads-kfold"
"196914","bedpe_to_vcf-split_reads-chrom"
"196915","bedpe_to_vcf-gridss-kfold"
"196916","bedpe_to_vcf-gridss-chrom"

$ ls *.log
stderr-196883.log  stderr-196886.log  stdout-196884.log  stdout-196887.log
stderr-196884.log  stderr-196887.log  stdout-196885.log  sv-channels.log
stderr-196885.log  stdout-196883.log  stdout-196886.log
$ cat sv-channels.log std*.log
...
```

**Comment by arnikz** [comment]
Same [issues](https://travis-ci.org/github/GooglingTheCancerGenome/sv-channels/builds/742934996) when using the pinned version of the `environment.yaml` (https://github.com/GooglingTheCancerGenome/sv-channels/commit/ef1ee77f433cb199b66f7f24762db46381a8b1c1).

**Comment by lsantuari** [comment]
@arnikz Thank you. The problem was due to odd window sizes. Now it should be ok.

**Comment by arnikz** [comment]
This time there are no FAILED jobs but still exits with non-zero. I noticed that not all jobs output `std[out,err].log` files in the same dir.

```
$ find -type f -name "std*.log"
./stderr-198521.log
./scripts/genome_wide/stderr-198532.log
./scripts/genome_wide/stdout-198586.log
./scripts/genome_wide/stdout-198567.log
./scripts/genome_wide/stdout-198537.log
./scripts/genome_wide/stderr-198534.log
./scripts/genome_wide/stdout-198582.log
./scripts/genome_wide/stdout-198565.log
./scripts/genome_wide/stderr-198533.log
./scripts/genome_wide/stderr-198536.log
./scripts/genome_wide/stderr-198558.log
./scripts/genome_wide/stderr-198565.log
./scripts/genome_wide/stderr-198530.log
./scripts/genome_wide/stdout-198566.log
./scripts/genome_wide/stdout-198530.log
./scripts/genome_wide/stdout-198532.log
./scripts/genome_wide/stdout-198568.log
./scripts/genome_wide/stdout-198559.log
./scripts/genome_wide/stderr-198559.log
./scripts/genome_wide/stderr-198535.log
./scripts/genome_wide/stdout-198583.log
./scripts/genome_wide/stderr-198581.log
./scripts/genome_wide/stderr-198537.log
./scripts/genome_wide/stderr-198586.log
./scripts/genome_wide/stdout-198531.log
./scripts/genome_wide/stderr-198568.log
./scripts/genome_wide/stderr-198585.log
./scripts/genome_wide/stdout-198536.log
./scripts/genome_wide/stdout-198585.log
./scripts/genome_wide/stdout-198581.log
./scripts/genome_wide/stdout-198529.log
./scripts/genome_wide/stderr-198582.log
./scripts/genome_wide/stderr-198529.log
./scripts/genome_wide/stdout-198533.log
./scripts/genome_wide/stderr-198566.log
./scripts/genome_wide/stderr-198584.log
./scripts/genome_wide/stdout-198534.log
./scripts/genome_wide/stderr-198583.log
./scripts/genome_wide/stderr-198531.log
./scripts/genome_wide/stderr-198567.log
./scripts/genome_wide/stdout-198584.log
./scripts/genome_wide/stdout-198558.log
./scripts/genome_wide/stdout-198535.log
./scripts/R/stderr-198592.log
./scripts/R/stdout-198593.log
./scripts/R/stderr-198594.log
./scripts/R/stdout-198594.log
./scripts/R/stdout-198595.log
./scripts/R/stdout-198592.log
./scripts/R/stderr-198593.log
./scripts/R/stderr-198595.log
./scripts/utils/stderr-198603.log
./scripts/utils/stdout-198601.log
./scripts/utils/stderr-198604.log
./scripts/utils/stderr-198601.log
./scripts/utils/stdout-198604.log
./scripts/utils/stdout-198603.log
./scripts/utils/stdout-198602.log
./scripts/utils/stderr-198602.log
./stdout-198520.log
./stdout-198519.log
./stderr-198522.log
./stdout-198521.log
./stdout-198523.log
./stderr-198523.log
./stdout-198522.log
./stderr-198519.log
./stderr-198520.log
```
Moreover, output files are [missing](https://travis-ci.org/github/GooglingTheCancerGenome/sv-channels/jobs/743177409#L14463). Most importantly, there are many (unnecessary) changes to the wrapper [`run.sh`](https://github.com/GooglingTheCancerGenome/sv-channels/compare/dev...iss64_fix#diff-d31ce0453051853c17ba2a5225b3d1bfab548e095bab0967d6acfd1b3ce1b35d) that makes debugging/merging a bit more difficult.

**Comment by arnikz** [comment]
Merged in 8ea1a1302b6945cb14928a8d15d4b87815e9ff06
